src/NativeBridge/stdafx.h

```diff
@@ -111,5 +107,14 @@ class StopWatch
 //
 // frequently used namespace aliases
 //
-namespace bp = boost::python;
-namespace np = boost::python::numpy;
\ No newline at end of file
+namespace bp = pybind11;
```

Comment:
Should `bp` be switched with `pb` here and throughout the rest of the code base to fully remove any references to boost python?

(Or switched to `py` which is used in some of the pybind examples. Though `py` seems to be a little too similar to the standard native python naming scheme)